### ISSUE TITLE
Recommended hardware

### ISSUE THREAD

**Issue by devinaconley** [monoDriveIO/documentation#182]
Add page with recommended hardware for
- desktop
- HIL / real time
- cloud

**Comment by devinaconley** [comment]
_Proposal_

Desktop
```
Intel Core i9-9900 or better
NVIDIA RTX 2080 Ti
32GB RAM or more
NVMe SSD
```

HIL / real-time
```
N x rack workstation (with above desktop specs)
```

Cloud
```
Kubernetes cluster
Prefer higher CPU speed over core count
Prefer higher GPU count
Prefer RTX GPUs
```

**Comment by citlaligm** [comment]
Closed with #252
